### PR TITLE
Update ios view with prompt

### PR DESCRIPTION
**Pull request by Falibur**

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<plugin xmlns="http://www.phonegap.com/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:rim="http://www.blackberry.com/ns/widgets" id="phonegap-plugin-barcodescanner" version="6.0.3">
+<plugin xmlns="http://www.phonegap.com/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:rim="http://www.blackberry.com/ns/widgets" id="phonegap-plugin-barcodescanner-ilost" version="6.0.3">
   <name>BarcodeScanner</name>
   <description>You can use the BarcodeScanner plugin to scan different types of barcodes (using the device's camera) and get the metadata encoded in them for processing within your application.</description>
   <license>MIT</license>
-  <repo>https://github.com/phonegap/phonegap-plugin-barcodescanner</repo>
-  <issue>https://github.com/phonegap/phonegap-plugin-barcodescanner/issues</issue>
+  <repo>https://github.com/ilostnl/phonegap-plugin-barcodescanner</repo>
+  <issue>https://github.com/ilostnl/phonegap-plugin-barcodescanner/issues</issue>
```
Comment (EddyVerbruggen) [inline plugin.xml]: This doesn't seem right :)

**Comment by Falibur** [comment]
Hmm, this is not what my intension was.. This PR should not be merged into master!

File: plugin.xml (hunk 1 above)
Comment (Falibur) [inline plugin.xml]: Completely agree :)
This PR is not my intension!

Comment (EddyVerbruggen) [inline plugin.xml]: Lol, happens to the best of us!

Comment (Falibur) [inline plugin.xml]: My apologies!
Can this be reverted?
This is my first fork from a public that lives in the public domain as well..

Comment (Falibur) [inline plugin.xml]: Thanks @EddyVerbruggen!

**Pull request closed without merging**